tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -97,30 +99,51 @@
 # .. role:: green
 # .. role:: red
 #
-# 1. Neuromag head coordinate frame ("head", :pink:`pink axes`)
-#      Defined by the intersection of 1) the line between the LPA
-#      (:red:`red sphere`) and RPA (:purple:`purple sphere`), and
-#      2) the line perpendicular to this LPA-RPA line one that goes through
-#      the Nasion (:green:`green sphere`).
-#      The axes are oriented as **X** origin→RPA, **Y** origin→Nasion,
+# 1. Neuromag/Elekta/MEGIN head coordinate frame ("head", :pink:`pink axes`)
+#      The head coordinate frame is defined through the coordinates of
+#      anatomical landmarks on the subject's head: Usually the Nasion (`NAS`_),
+#      and the left and right preauricular points (`LPA`_ and `RPA`_).
+#
+#      Concretely, the head coordinate frame is defined by the intersection of
+#
+#      1. the line between the LPA (:red:`red sphere`) and RPA
+#         (:purple:`purple sphere`), and
+#      2. the line perpendicular to this LPA-RPA line one that goes through
+#         the Nasion (:green:`green sphere`).
+#
+#      The axes are oriented as **X** origin→RPA, **Y** origin→NAS,
 #      **Z** origin→upward (orthogonal to X and Y).
 #
-#      .. note:: This gets defined during the head digitization stage during
-#                acquisition, often by use of a Polhemus or other digitizer.
+#      .. note:: The required 3D coordinates for defining the head coordinate
+#                frame (NAS, LPA, RPA) are measured at a stage separate from
+#                the MEG data recording. There exist numerous devices to
+#                perform such measurements, usually called "digitizers". For
+#                example, see the devices by the company `Polhemus`_.
+#
+#      .. note:: Other MEG manufacturers may have different definitions of the
+#                coordinate head frame, a good overview can be seen in the
+#                section "How are the different head and MRI coordinate systems
+#                defined?" in the FAQ of the FieldTrip toolbox WIKI.
```

Comment:
Too many `.. note`s become difficult to read. Either make some/all of these normal paragraphs, or move them into `.. sidebar::`, which renders like the gray-greenish box on the right here

http://mne-tools.github.io/dev/auto_tutorials/intro/plot_20_events_from_raw.html#converting-a-stim-channel-signal-to-an-events-array